packages/dashboard-core-plugins/src/linker/LinkerLink.scss

```
@@ -12,89 +12,151 @@ $dash-delete-color-1: $danger;
 $dash-delete-color-2: mix($danger, $content-bg, 25%);
 $dash-size: 10;
 
-.linker-overlay svg {
-  .link-dot,
-  .link-triangle {
-    fill: $dash-color-1;
+.linker-overlay {
+  .btn-fab {
+    pointer-events: all;
+    position: absolute;
+    min-width: 1rem;
+    min-height: 1rem;
+    padding: 0.25rem 0.45rem;
+    z-index: 30;
   }
```

```suggestion
  .btn-fab {
    position: absolute;
    display: flex;
    justify-content: center;
    align-items: center;
    pointer-events: all;
    min-width: $input-height;
    min-height: $input-height;
    height: $input-height;
    width: $input-height;
    line-height: $font-size-lg; // vertical centering for text inside fa-layer
    padding: 0;
    margin: 0;
    z-index: 30;
    box-shadow: $box-shadow;
    .fa-layers {
      display: inline-flex;
      justify-content: center;
      b {
        // comparison operator
        font-size: $font-size-lg;
      }
    }
  }
```

- Button insides weren't centered
- Try to use existing variables instead of hardcode values
- Increased text size
- Added box-shadow